docs/src/user/manual/data-upload/panel.md

```diff
@@ -1,2 +1,204 @@
-# Panel Overview
-<!-- To Be Populated -->
+# Data Upload Page Documentation
+
+## Description
+
+The Data Upload Page is a key component for managing and uploading your data. This page provides an intuitive interface for adding new data and customizing your view. It also includes a map with interactive features to help you visualize spatial data.
+
+### Data upload
+![Data upload](./img/panel-1.png)
+
+
+1. **Search area**: The central feature of the Data Upload Page is the data search area, where you can search for specific areas within your dataset. This search field allows you to enter keywords or details related to your records to locate specific areas.
+
+2. **Select property**: The Property Selection Dropdown is a vital tool for categorizing your data. You can choose from existing property categories to associate your data with a specific property. This helps organize and categorize your records effectively.
+
+3. **Create a new propertty**: If the property you want to associate your data with doesn't exist in the dropdown, you can create a new property. Click the `CREATE A NEW PROPERTY` button to add a new property category. You'll be prompted to enter a name and details for the new property, and it will be added to the dropdown for future use.
```

Comment:
```suggestion
3. **Create a new property**: If the property you want to associate your data with doesn't exist in the dropdown, you can create a new property. Click the `CREATE A NEW PROPERTY` button to add a new property category. You'll be prompted to enter a name and details for the new property, and it will be added to the dropdown for future use.
```